admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+
+The following information is not an in-depth guide, but more of a high-level overview of each type.
+
+Anonymous
+---------
+
+- Is not a regular user.
+- Has access to specific content made available via public links.
+- Can be password-protected (optional, enforced, policy-enforced).
+- Can have an expiration date (optional, enforced, enforced dependent on password).
```

Comment:
sub-item of public links above, needs indentation

Reply:
I've got that done locally.